### PR TITLE
Use R version aliases

### PR DESCRIPTION
**Pull request by schloerke**
* Init using R version aliases. 
* Install R on docker to get the installed version

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -27,125 +27,110 @@ jobs:
...
         SHORT_SHA=${SHA:0:7}
-        echo $SHORT_SHA
+        echo "sha: $SHORT_SHA"
         echo "::set-output name=sha::$SHORT_SHA"
-
+    - name: Should install
+      id: can-build
+      if: ${{ !( steps.short.outputs.r-version == '3.5' && matrix.docker.type == 'bionic') }}
+      run: |
+        echo "Did this step run?"
```
Comment (cpsievert) [inline .github/workflows/docker.yml:83]:
```suggestion
        echo "If this step did not run, then we're knowingly unable to install this version of R on this Linux distro"
```

File: Docker/ubuntu_ssp/Dockerfile [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
+ARG RELEASE=focal
 ARG EXTRA_BASE_TAG=
 FROM rstudio/shinycoreci:base-${R_VERSION}-${RELEASE}${EXTRA_BASE_TAG}
 
-ARG R_VERSION=3.6
-ARG RELEASE=bionic
+ARG R_VERSION=4.1
+ARG RELEASE=focal
 ARG EXTRA_BASE_TAG=
 ARG AWS_BUILD_MACHINE=ubuntu-14.04
```
Comment (cpsievert) [inline Docker/ubuntu_ssp/Dockerfile:19]:
```suggestion
```

File: Docker/ubuntu_ssp/Dockerfile [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
 
-RUN apt-get update && apt-get install -y \
-  libssl1.0.0 \
-  psmisc
-
 # =====================================================================
 # Shiny Server
 # =====================================================================
```
Comment (cpsievert) [inline Docker/ubuntu_ssp/Dockerfile:24]: Add build website link

**Comment by schloerke** [comment]
Once a centos7 docker succeeds, then this PR may be merged

**Pull request merged**